app/helpers/mortgage_calculator/land_and_buildings_transaction_taxes_helper.rb

```diff
@@ -52,5 +64,21 @@ def maximum_band(num)
     def formatted_currency(num)
       number_to_currency(num, precision: 0)
     end
+
+    def standard_lbtt(price)
+      calculator.new(price: price, buyer_type: 'isNextHome').tax_due
+    end
+
+    def ftb_lbtt(price)
```

Comment:
This method and the previous are basically doing the same thing, maybe we could refactor to something like:
```ruby
lbbt_tax(price: price, type: buyer_type)
```
?